modules/task.js

```diff
@@ -0,0 +1,92 @@
+import Element from './element.js';
+
+const elementObject1 = new Element();
+let editId = 0;
+let i = null;
+export default class Task {
+  constructor() {
+    this.taskList = [];
+  }
+
+  addTask = () => {
+    if (editId !== 0 && i !== null) {
+      this.updateTask(elementObject1.titleInput.value, i);
+    } else {
+      const tasks = {
+        title: elementObject1.titleInput.value,
+        completed: false,
+      };
+      this.taskList.push(tasks);
```

Comment:
- [ ] Good job so far 👏 👏 So far, most of the functionalities are working well, However, I realized that upon creating a new Task object, you forgot to add an important property which is `id` or `index`. Note that this is different from the index of the array in which task objects are stored because apart from being the individual property of the Task object, It counts starting from 1, unlike the array index 😃 So that means the first task that should be added, its index property should be set to 1 and then we increment from there 😉
![image](https://user-images.githubusercontent.com/90222110/228964246-0b48e723-d9af-4b00-a70c-a419ee0b29c0.png)

Please add that index property and make sure that it works as stated in the project requirements